streamer/parser/a11y-metadata-parsing.md

```diff
@@ -0,0 +1,271 @@
+# Parsing Accessibility Metadata
+
+The goal of this document is to provide directions that each implementation of Readium can follow when parsing schema.org Accessibility Metadata in EPUB publications.
+
+When parsing a publication in the streamer we always use the most complex form for each metadata to harmonize our output.
+
+Related Repository: [Readium Web Publication Manifest](https://github.com/readium/webpub-manifest)
+
+References: 
+
+- http://kb.daisy.org/publishing/docs/metadata/schema-org.html
+- http://kb.daisy.org/publishing/docs/metadata/evaluation.html
+- https://www.w3.org/wiki/WebSchemas/Accessibility
```

Comment:
I would suggest adding:
* https://www.w3.org/TR/pub-manifest/#accessibility
* https://w3c.github.io/publ-a11y/UX-Guide-Metadata/techniques/schema-org.html

Reply:
Oh, and of course:
https://www.w3.org/Submission/epub-a11y/#sec-disc-package

(even though it is mentioned inside the document, it is good to have the top-level reference too)